libraries/AP_HAL_ESP32/WiFiDriver.cpp

```diff
@@ -50,7 +51,7 @@ void WiFiDriver::_begin(uint32_t b, uint16_t rxS, uint16_t txS)
     if (_state == NOT_INITIALIZED) {
         initialize_wifi();
 
-	if (xTaskCreatePinnedToCore(_wifi_thread, "APM_WIFI1", Scheduler::WIFI_SS1, this, Scheduler::WIFI_PRIO1, &_wifi_task_handle,0) != pdPASS) {
+	if (xTaskCreatePinnedToCore(_wifi_thread, "APM_WIFI1", Scheduler::WIFI_SS1, this, Scheduler::WIFI_PRIO1, &_wifi_task_handle, 0) != pdPASS) {
```

Comment:
I wonder wether the wifi thread would be faster if it was put onto CPU 1 , not cpu 0 as it is here, maybe u could test this ?  not a stopper for this PR tho.

Reply:
I didn't really look how the _wifi_thread() is working. I should check my ESP8266 code because on a normal Unix machine I'm using select() , kqueue() or epoll(). That hal.scheduler->delay_microseconds() is not how it's done on the bigger OSes. Not sure about ESP32